src/drive/web/modules/actions/index.jsx

```diff
@@ -85,7 +85,7 @@ export const download = ({ client }) => {
               }}
               left={<Icon icon="download" />}
             >
-              {t('toolbar.menu_download_folder')}
+              {t('SelectionBar.download')}
```

Comment:
I see no change to locale files here, am I missing something ?

Reply:
The fact that the previous one was not the right one in this context. And that `SelectionBar.download` already exists